hmt_escrow_js/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "hmt-escrow-form-js",
+  "version": "1.0.0",
+  "description": "JS File for HMT-ESCROW-FORM",
+  "main": "server.js",
```

Comment:
The main would be the main entrypoint for the package, so probably an `index.js` file or something that exports `{ Job, EthInterface }` right?

It'd be good to see in your form something like:

```
import { Job } from 'hmt-escrow';
Job(..)
```

Reply:
fixed